app/views/races/_new.slim

```diff
@@ -58,9 +58,8 @@
           .tab-pane.text-light#randomizer-content role='tabpanel' aria={labelledby: 'randomizer-content-tab'}
             form#randomizer-races-form
               .form-group
-                label for='randomizer-seed-input' Seed
-                input.form-control#randomizer-seed-input type='text' placeholder='Seed' required=true
-                .valid-feedback Looks good!
-                .invalid-feedback Required.
+                label for='randomizer-seed-input' Seed name (optional)
+                input.form-control#randomizer-seed-input type='text' placeholder='Seed name'
+                small.form-text.text-muted You'll be able to attach a seed file once the race is created.
```

Comment:
How come you went with attaching after creation instead of requiring it to proceed and having it be unchanging?

Reply:
This is one of my philosophies about building things (see: splitsdb vs splits.io). i.e. don't require any upfront work to get going, and always let people change their mind.

Specific to this scenario, I was thinking people would feel "I want to do a randomizer!", then get people to join them, and actually generating a seed file would be the last thing on their mind. So I don't want to put a hurdle up in front of people that says "people can't join you until you finish your chores".

I also want to allow people to change their mind -- maybe they were going to share the file in some other way or not at all, then once everyone has joined the race (perhaps someone they don't know joins) they decide to share it using Splits.io. the alternative would be to force them to recreate the race or settle for some other way of sharing files.

Reply:
Ok, that does make a lot of sense.  100% agree with the sentiment of creating the race before even thinking about creating the seed file.

I would like to make an argument for having a seed file be present for a randomizer race to start though.  I think being able to go back to a race and be able to download their seed file is a cool concept and having it be there for historical purposes also sounds nice.  Obviously you could upload a blank file or something, but it would encourage sharing the seed file for others to be able to see.

Reply:
I agree having the seed file consistently be on Splits.io for historical purposes is a great goal. I agree we should encourage it. I disagree that we should force it.

If we force users to upload a seed file and they don't want to / it doesn't make sense to, they will either choose not to race using Splits.io at all (which we need to avoid early on for racing to catch on through the network effect), or like you said they'll upload some random file to get past the hurdle, which by itself is its own hurdle -- both in the fact that you have to go through the effort to do so, but also because you have to learn that it's possibility, and not that you've hit a brick wall by not having a file to upload.

My biggest worry is that you and I personally don't understand the entirety of the randomizer community, and for one reason or another some communities straight up won't be uploading files at all. Maybe it's the standard that only the seed name is shared and seed files are generated locally, or maybe the randomizer program has its own built-in share feature that's 1000% better than what we can provide, or maybe even the game itself allows you to input the seed, so there's literally no file to be shared at all (see: Crypt of the Necrodancer).

I want to make all races reproducible for historical purposes. Not by forcing users to upload seed files but by making the experience of sharing your seed file through us far better than any other way of doing so; and for communities where sharing a seed file doesn't even make sense, talk to those communities (later, after we deploy this) and figure out if there is a low-friction way to preserve that history if the seed name field isn't enough.

Reply:
Those are all very compelling arguments.  I can't really disagree with any of it so I retract my desire to force a seed file :+1: